hal/include/HAL/SerialPort.h

```diff
@@ -8,13 +8,11 @@
 #pragma once
 
 #include <stdint.h>
+#include "HAL/Types.h"
```

Comment:
Since this is now including types, will you remove the <stdint.h> include, since that is in types.